cli/config/config.go

```diff
@@ -150,3 +155,24 @@ func (m *Metadata) ExpandEnv() {
 		m.Tags[i] = os.ExpandEnv(v)
 	}
 }
+
+// SyncCapabilities sync capabilities if no settings
```

Comment:
Would be great to expand this comment further. E.g.
```suggestion
// SyncCapabilities uses the project capabilities if no settings have been defined in the suites.
```

Reply:
Sure. Updating